packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerBody.tsx

```diff
@@ -24,25 +21,26 @@ export function MomentTemplateBannerBody({
 }: MomentTemplateBannerBodyProps): JSX.Element {
     const styles = getStyles(highlightedTextSettings);
 
+    const isTabletOrAbove = useMediaQuery(from.tablet);
+
+    const bannerBodyMobile = createBannerBodyCopy(
+        mobileContent.paragraphs,
+        mobileContent.highlightedText,
+        styles,
+    );
+    const bannerBodyTabletAndAbove = createBannerBodyCopy(
+        mainContent.paragraphs,
+        mainContent.highlightedText,
+        styles,
+    );
+
     return (
         <div css={styles.container}>
-            <Hide above="tablet">
-                {createBannerBodyCopy(
-                    mobileContent.paragraphs,
-                    mobileContent.highlightedText,
-                    styles,
-                )}
-            </Hide>
-
-            <Hide below="tablet">
-                {createBannerBodyCopy(mainContent.paragraphs, mainContent.highlightedText, styles)}
-            </Hide>
+            {isTabletOrAbove ? bannerBodyTabletAndAbove : bannerBodyMobile}
```

Comment:
small optimisation - could we call `createBannerBodyCopy` inline here to avoid creating both elements above?

Reply:
Good shout!